src/components/error-page/error-page-slide.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+import SlideError from "components/shared/slide-error";
+import ICON_ERROR from "assets/images/icon_error.png";
+
+const ErrorPageSlide = ({ handle }) => (
+<SlideError title="Uh oh! Something went wong." image={null} >
+    <img src={ICON_ERROR} className="error-img" alt="error-img" />
+    <p className="error-description">
+    There was a problem with your upload. Please <a className="error-description__link" href="#"> contect support </a> about your issue.
```

Comment:
Do we specify how they should contact support?  Like by email or whatever?

Reply:
Can we have there link "contact support by email" and href will be email... but what email? support@oyster.ws for example or what?

Reply:
yeah support@oyster.ws is fine.  Alex F set that e-mail up for us.